bin/install-forch.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/ash
+
+set -euo pipefail
```

Comment:
install_forch.sh  (underscore not dash in filename)

Reply:
renamed file